CONTRIBUTING.md

```diff
@@ -15,3 +15,18 @@ If you want to contribute to the Beats project, you can start by reading
 the [contributing guidelines](https://www.elastic.co/guide/en/beats/devguide/current/beats-contributing.html)
 in the _Beats Developer Guide_.
 
+## Dev Tools
+
+First of all, check that you have following tools installed in your local computer:
+
+- [`mage`](https://magefile.org/#installation)
```

Comment:
`mage` must be installed from the `vendor` directory to ensure that the proper version is installed. All of the `make` commands that delegate to `mage` targets automatically install `mage` for you, but if you want to manually setup `mage` then you can run `make mage` which will put a `mage` binary into `$GOPATH/bin`. Windows users can run any of the `.\make.bat` files in the project; they do the same thing as `make mage`.

Reply:
Cool! I change this to reflect the usage of vendor to install `mage`, using `make mage` (although the command name sounds weird).

@dedemorton, I agree with you that having it in the guide is must, but I also see the value having it in project's README/CONTRIBUTING. WDYT?

Reply:
@mdelapenya I see your point, but I think all the info about setting up the dev environment should live in one place so that we have a single source of truth. Otherwise we end up duplicating content, it gets out of sync (always!), and users are not sure where to go for the information that they need. That's why we decided to consolidate all the content in the dev guide.

Reply:
Cool then. I also agree in having one single point of (failure) truth with docs! 😂